.eslintrc.js

```diff
@@ -0,0 +1,13 @@
+module.exports = {
+  env: {
+    browser: true,
+    es2021: true,
```

Comment:
Based on the comment on this PR: https://github.com/snyk/topcoat-core/pull/511
`eslint` should be warning us about the usage of `module.` in this file right? 
Am I missing something? Maybe I misunderstood the comment on the other PR 😄

Reply:
Indeed it should be showing an error, however it does NOT! 😓 

Weird. Do you know why there is a difference?.... Ah. I think I know. There's no `package.json`. It is failing when trying to run eslint in VSCode:

```
To use ESLint please install eslint by running npm install eslint in the workspace folder topcoat-public
or globally using 'npm install -g eslint'. You need to reopen the workspace after installing eslint.
```

I guess we need a package.json or to install all of the packages globally 🙄 
```
eslint
eslint-plugin-vue
```

I guess I could try adding a package.json and see if it is also not picked up by the builder... but it's getting mucky!

Reply:
Okay, so I've added a package.json, but it does get added to the modules folder when it is re-built

<img width="235" alt="Screenshot 2022-10-19 at 16 26 57" src="https://user-images.githubusercontent.com/1307818/196719582-69c8f167-9f1e-4066-976c-abacee357890.png">

I'm not sure what sort of consequences this has to be honest.

Reply:
It doesn't seem to add any of the dotfiles, which is interesting